src/components/GameView/ChooseWhetherToCounterDialog.vue

```diff
@@ -0,0 +1,123 @@
+<template>
```

Comment:
Counterpart to the above `ChooseTwoDialog`. This is the one that first pops up to ask if you'd like to counter, then we switch to the one to which two to counter with if you do choose to counter.